components/skus/browser/rs/cxx/src/httpclient.rs

```diff
@@ -85,7 +85,9 @@ impl From<ffi::HttpResponse<'_>> for Result<http::Response<Vec<u8>>, InternalErr
                         )
                     })?;
 
-                    response.headers_mut().ok_or(InternalError::BorrowFailed)?.insert(key, value);
```

Comment:
unrelated fix, the error used here was inaccurate and the proper error will be raised below when the builder is completed